lecture/forms.py

```
@@ -6,7 +6,7 @@
 class LectureForm(forms.ModelForm):
     class Meta:
         model = Lecture
-        fields = ['course','module','title','description','video_url','content','order']
+        fields = ['module','title','description','video_url','content','order']
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)